src/styles/blocks/calendar.scca

```diff
@@ -0,0 +1,49 @@
+.calendar {
+  width: 7 * ($cell-size + $gap) + 2 * $padding;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  padding: $padding;
+
+  &__day {
+    @include center();
+    align-items: center;
+    @include cell($cell-size);
+    transition: 0.5s;
+
```

Comment:
Transition by default used for `all`. Set only for background and transform.
`transition: background-color 0.5s, transform 0.5s;` 
Move duration to variable